lib/parse.js

```diff
@@ -490,8 +492,10 @@ function tokenizer($TEXT, filename, html5_comments, shebang) {
     }
 
     function read_num(prefix) {
-        var has_e = false, after_e = false, has_x = false, has_dot = prefix == ".";
+        var has_e = false, after_e = false, has_x = false, has_dot = prefix == ".", isBigInt = false;
         var num = read_while(function(ch, i) {
+            if (isBigInt) return false;
```

Comment:
Could we use snake_case here? is_big_int please :)

Reply:
Yeap ;)